### PR TITLE
LibGfx+icc: Parse and print rendering intent from ICCProfile header

### PR DESCRIPTION
**Pull request by nico**
…and add rendering intent dumping to `icc`.

### CODE REVIEW

File: Userland/Libraries/LibGfx/ICCProfile.cpp [hunk 1]
```diff
@@ -255,6 +271,22 @@ char const* color_space_name(ColorSpace color_space)
     }
 }
 
+char const* rendering_intent_name(RenderingIntent rendering_intent)
```
Comment (Hendiadyoin1) [inline Userland/Libraries/LibGfx/ICCProfile.cpp:274]: Why no StringView?
(Is the NULL state ever supposed to be reached without the world burning around this function?)

Comment (nico) [reply Userland/Libraries/LibGfx/ICCProfile.cpp:274]: See discussion here: https://github.com/SerenityOS/serenity/pull/16724#discussion_r1059401594

(No.)

**Comment by awesomekling** [comment]
In the future, please title PRs appropriately and keep unrelated typo fixes separate :)

**Comment by nico** [comment]
What!

**Pull request merged**